Amazon.QLDB.Driver/session/SessionPool.cs

```diff
@@ -82,24 +80,12 @@ public T Execute<T>(Func<TransactionExecutor, T> func, RetryPolicy retryPolicy,
         }
 
         /// <summary>
-        /// Dispose the session pool and all sessions.
-        /// </summary>
-        public void Dispose()
-        {
-            this.isClosed = true;
-            while (this.sessionPool.Count > 0)
-            {
-                this.sessionPool.Take().Close();
-            }
-        }
-
-        /// <summary>
-        /// <para>Get a <see cref="IQldbSession"/> object.</para>
+        /// <para>Get a <see cref="QldbSession"/> object.</para>
```

Comment:
Why was this changed?

Reply:
Because IQldbSession does not exist in the code.